content-src/lib/filters.js

```diff
@@ -0,0 +1,28 @@
+function createFilter(definition) {
+  return function(item) {
+    let result = true;
+    definition.forEach(test => {
+      if (!test(item)) {
+        result = false;
+      }
+    });
+    return result;
+  };
+}
+
+const URL_FILTERS = [
+  item => !!item.url,
+  item => item.url && item.url.length < 100,
```

Comment:
This is kind of a stop-gap until we implement the POST-style requests. I'll make a note